src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -504,15 +506,12 @@ public IEnumerable<IRenderPackage> BuildRenderPackage()
         /// <summary>
         /// Checks if manipulator is enabled or not. Manipulator is enabled 
         /// only if node is not frozen or not setup as partially applied function 
-        /// or Run setting is set to Automatic.
         /// </summary>
         /// <returns>True if enabled and can be manipulated.</returns>
         public bool IsEnabled()
         {
             if (Node.IsFrozen) return false;
 
-            if (Node.CachedValue.IsNull) return false;
```

Comment:
This check was to prevent manipulation when the Node's value becomes null for some reason. How are we ensuring that manipulators are not enabled when the preview value turns null and evaluation was not triggered to destroy the manipulator.